commands/git/git-nb.md

```diff
@@ -0,0 +1,13 @@
+### Create a new branch
```

Comment:
I feel like, file name should not be in short forms, let's change it to `git-new-branch` so it will be easy to identify content by filename itself